### ISSUE TITLE
Uncertified device?

### ISSUE THREAD

**Issue by techdudepro** [opengapps/opengapps#434]
I apologize if this is the wrong place to post this, I'm assuming that this has to do with opengapps.

In Play Store >> Settings (under "About") there is info regarding device certification. Here is Google's help page with more info: 

https://support.google.com/googleplay/answer/7165974?visit_id=0-636210062546690980-729244030&p=uncertified_device&rd=1

I am curious if this would be an issue for custom ROMs and gapps flashed on them. As in will Google make it hard to do more unauthorized tinkering (similar to their "secure boot" feature introduced in Nougat). This is more of an informative heads-up?

Steps to reproduce:

1. Completely wipe, format everything
2. Install TWRP v. 3.0.2
3. Install CyanogemMod 13 using the new TWRP
4. Flash pico opengapps using the new TWRP

If it makes any difference, I have root enabled, and have tweaked some using L Speed (search in Google Play).

**Comment by NicholasBuse** [comment]
First, This discussion should be taking place on XDA. (but see my follow-up in the next comment)

Please read the [FAQ ](https://github.com/opengapps/opengapps/wiki/FAQ#2-im-having-a-problem-with-my-open-gapps-install-what-information-do-you-need-to-help-me)and the support instructions mentioned. You need to use [XDA Forum](http://forum.xda-developers.com/android/software/Open-GApps-t3098071) for support requests, directly opening GitHub issues is not appreciated.

PS: Please remember to at least include your installation debug log and (if applicable) a logcat when requesting support.

**Comment by NicholasBuse** [comment]
TL;DR; This probably is **NOT** an OpenGApps issue...

My 6P shows the same information in the PlayStore, I have **Stock ROM** + TWRP + SuperSU (therefore my bootloader is unlocked).

I am sure that it is similar to the mechanism (SafetyNet) in PlayServices that detect devices that are not compliant with SafetyNet This makes games like Pokemon Go and apps like Android Pay not work. I will try backing off SuperSU, then TWRP to see if I can get my device to be compliant. 

My guess is what you found is Google's way of showing you that your device is SafetyNet compliant. I believe that even on my device removing SuperSU and TWRP will **NOT** be enough since currently SafetyNet requires a locked bootloader too...

**Comment by techdudepro** [comment]
My apologies... I should have used XDA first, as in the FAQ. And kept my installation log (I deleted it because I had to issues).

Sorry/thanks for your time! I'll close this to help de-clutter.
